builder-nodejs8x/Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM jenkinsxio/builder-base:1
 RUN curl -f --silent --location https://rpm.nodesource.com/setup_8.x | bash - && \
   yum install -y nodejs gcc-c++ make bzip2 GConf2 gtk2 chromedriver chromium xorg-x11-server-Xvfb
 
-RUN npm i -g watch-cli vsce typescript && npm up -g npm
+RUN npm i -g watch-cli vsce typescript npm@6.5.0 && [ "6.5.0" == "$(npm -v)" ]
```

Comment:
I think you can drop the ` && [ "6.5.0" == "$(npm -v)" ]` part, since that is always true